doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,306 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+    :align: left
+
+The job page allows you to see tasks and actors broken down by their states. 
+Tasks and actors are grouped and nested by default. You can see the nested entries by clicking the expand button.
+
+Tasks/actors are grouped and nested by the following criteria.
+
+- All tasks and actors are grouped together, and you can view individual entries by expanding the corresponding row.
+- Tasks are grouped by their ``name`` attribute (e.g., ``task.options(name="<name_here>").remote()``).
+- Child tasks (nested tasks) are nested under their parent task's row.
+- Actors are grouped by their class name.
+- Child actors (actors created within an actor) are nested under their parent actor's row.
+- Actor tasks (remote methods within an actor) are nested under the actor for the corresponding actor method.
+
+.. note:: 
+
+  Ray dashboard can only display/retrieve up to 10K tasks at a time. If there are more than 10K tasks from your job,
+  they are unaccounted. The number of unaccounted tasks is available from the task breakdown.
+
+Task Timeline
+~~~~~~~~~~~~~
+
+The :ref:`timeline API <ray-core-timeline>` is available from the dashboard. 
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/profile-button.png
+    :align: center
+
+First, you can download the chrome tracing file by clicking the download button.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/profile_drag.png
+    :align: center
+
+Second, you can use tools like ``chrome://tracing`` or the `Perfetto UI <https://ui.perfetto.dev/>`_ and drop the downloaded chrome tracing file. We will use the Perfetto as it is the recommendation way to visualize chrome tracing files.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/timeline.png
+    :align: center
+
+Now, you can see the timeline visualization of Ray tasks and actors. There are Node rows (hardware) and Worker rows (processes). 
+Each worker rows display a list of events (e.g., task scheduled, task running, input/output deserialization, etc.) happening from that worker over time.
+
+Ray Status
+~~~~~~~~~~
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/ray-status.png
+    :align: center
+
+The job page displays the output of the helpful CLI tool ``ray status``, which shows the autoscaler status of the Ray cluster.
+
+The left page shows the autoscaling status, including pending, active, and failed nodes. 
+The right page displays the cluster's demands, which lists demanded resources that cannot be scheduled to the cluster at the moment. This page is useful for debugging resource deadlocks or slow scheduling.
+
+.. note:: 
+
+  The output shows the aggregated information across the cluster. If you run more than one job, the demands could be wrong.
```

Comment:
Instead of saying it's "wrong", maybe we should say something like "some of the demands may come from other jobs."